### PR TITLE
feat(worker): add worker commands

### PR DESCRIPTION
**Pull request by wass3rw3rk**
for https://github.com/go-vela/cli/pull/430

### CODE REVIEW

File: content/reference/cli/worker/update.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+The following parameters are used to configure the command:
+
+| Name              | Description                      | Environment Variables                           |
+| ----------------- | -------------------------------- | ----------------------------------------------- |
+| `worker.address`  | address of the worker            | `VELA_WORKER_ADDRESS`, `WORKER_ADDRESS`         |
+| `worker.hostname` | hostname of the worker           | `VELA_WORKER_HOSTNAME`, `WORKER_HOSTNAME`       |
+| `active`          | current status of the worker     | `VELA_WORKER_ACTIVE`, `WORKER_ACTIVE`           |
+| `build-limit`     | build limit for the worker       | `VELA_WORKER_BUILD_LIMIT`, `WORKER_BUILD_LIMIT` |
+| `routes`          | route assignment for the worker  | `VELA_WORKER_ROUTES`, `WORKER_ROUTES`           |
```
Comment (ecrupper) [inline content/reference/cli/worker/update.md]: I don't think it's possible to adjust the build limit via the API: https://github.com/go-vela/server/blob/7eda80aa14a51ef82dba73cda9f2556f48e346d7/api/worker.go#L311-L324. Also, even though it technically adjusts routes, we don't actually go in and change it on that worker, so I wonder if advertising it in the docs is kind of misleading?

File: content/reference/cli/worker/update.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+To install the CLI, please review the [installation documentation](/docs/reference/cli/install/).
+
+To setup the CLI, please review the [authentication documentation](/docs/reference/cli/authentication/).
+{{% /alert %}}
+
+#### Request
+
+```sh
+$ vela update worker --worker.hostname MyWorker --worker.address newaddress.example.com
```
Comment (ecrupper) [inline content/reference/cli/worker/update.md:56]: Should we advertise the new way of just using worker address?

File: content/reference/cli/worker/view.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+{{% /alert %}}
+
+## Parameters
+
+The following parameters are used to configure the command:
+
+| Name                        | Description                                                               | Environment Variables                                         |
+| --------------------------- | ------------------------------------------------------------------------- | ------------------------------------------------------------- |
+| `worker.hostname`           | hostname of the worker                                                    | `VELA_WORKER_HOSTNAME`, `WORKER_HOSTNAME`                     |
+| `worker.registration.token` | show the registration token for the worker (requires Vela Platform Admin) | `VELA_WORKER_REGISTRATION_TOKEN`, `WORKER_REGISTRATION_TOKEN` |
```
Comment (ecrupper) [inline content/reference/cli/worker/view.md]:
```suggestion
| `worker.registration.token` | generate a registration token for the worker (requires Vela Platform Admin) | `VELA_WORKER_REGISTRATION_TOKEN`, `WORKER_REGISTRATION_TOKEN` |
```

Maybe?

**Review by ecrupper — CHANGES_REQUESTED** [review]
Nice — couple things I noticed

File: content/reference/cli/worker/update.md (hunk 2 above)
Comment (wass3rw3rk) [reply content/reference/cli/worker/update.md:56]: Good call

File: content/reference/cli/worker/view.md (hunk 3 above)
Comment (wass3rw3rk) [reply content/reference/cli/worker/view.md]: i'm cool with that

File: content/reference/cli/worker/update.md (hunk 1 above)
Comment (wass3rw3rk) [reply content/reference/cli/worker/update.md]: i mean it does change the value, the value is just not being used. how about we mark those values as `(unused)`?

Comment (wass3rw3rk) [reply content/reference/cli/worker/update.md]: Oops, my bad. For that specific value, you are correct. Even for the other unused values, i think we want to mark them somehow.

Comment (wass3rw3rk) [reply content/reference/cli/worker/update.md]: addressed

File: content/reference/cli/worker/update.md (hunk 2 above)
Comment (wass3rw3rk) [reply content/reference/cli/worker/update.md:56]: addressed

File: content/reference/cli/worker/add.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+To install the CLI, please review the [installation documentation](/docs/reference/cli/install/).
+
+To setup the CLI, please review the [authentication documentation](/docs/reference/cli/authentication/).
+{{% /alert %}}
+
+#### Request
+
+```sh
+$ vela add worker --worker.hostname MyWorker --worker.address myworker.example.com
```
Comment (plyr4) [inline content/reference/cli/worker/add.md]: minor, should we include protocol? 
```suggestion
$ vela add worker --worker.hostname MyWorker --worker.address https://myworker.example.com
```

Comment (plyr4) [reply content/reference/cli/worker/add.md]: meh, not necessary, that should be inferred

**Review by ecrupper — APPROVED** [review]

**Review by plyr4 — APPROVED** [review]

**Review by timhuynh94 — APPROVED** [review]

**Pull request merged**